Backpack/BottomSheet/Classes/BottomSheet.swift

```diff
@@ -100,9 +100,30 @@ public final class BottomSheet: NSObject {
     ///   - animated: Animated or not.
     ///   - completion: Completion closure called after presentation animation.
     @objc(presentInViewController:animated:completion:)
-    public func present(in viewController: UIViewController, animated: Bool, completion: (() -> Void)?) {
+    public func present(in viewController: UIViewController, animated: Bool, completion: (() -> Void)? = nil) {
         viewController.present(viewControllerToPresent, animated: animated, completion: completion)
     }
+    
+    /// This method allows presenting a new bottom sheet on top of a previously existing one.
+    /// The previous bottom sheet is automatically moved to the initial position, its scroll view content
+    /// inset is reset, and the new bottom sheet won't add any more alpha to the backdrop view.
+    /// - Parameters:
+    ///   - bottomSheet: The new bottom sheet to present.
+    ///   - animated: Animated or not.
+    ///   - completion: Completion closure called after the presentation animation.
+    @objc(presentBottomSheet:animated:completion:)
+    public func present(_ bottomSheet: BottomSheet, animated: Bool, completion: (() -> Void)? = nil) {
+        if let scrollView = floatingPanelController.scrollView {
```

Comment:
The calls in this method that pass `animated:` don't respect the `animated` argument passed ot this method. It feels like they should?